LBD2024G06TP1Script.sql

```diff
@@ -192,54 +247,78 @@ VALUES (1, 123456789, 'Juan', 'Perez', 'A'),
        (19, 987654123, 'Ezequiel', 'Gimenez', 'A'),
        (20, 456987321, 'Valeria', 'Peralta', 'A');
 
+-- Populate the table DUEÑOS
```

Comment:
Empleados and dueños are exactly the same names! Plese change it so we dont have confusions in the future :(